js/ol5-sidebar.js

```diff
@@ -0,0 +1,141 @@
+// JS imports
+import Control from 'ol/control/Control';
```

Comment:
as I mentioned before this is invalid ES5 code and should not be shipped in the current state